kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/form/render/AbstractFormRenderer.java

```diff
@@ -183,6 +184,33 @@ public String renderCase(String containerId, CaseDefinition caseDefinition, Form
         return finalOutput;
     }
 
+    private String getLocalDateWithoutTime() {
+        StringBuilder function = new StringBuilder();
+
+        function.append("function ")
```

Comment:
I'd move this function declaration to the `kieserver-ui.js` instead of adding it to the template every time.

BTW noticed that this is only added to process forms, not tasks forms, so task forms are breaking because the function isn't present. So another reason to make it available at the `kieserver-ui.js`

Reply:
Done